ember-css-transitions/package.json

```diff
@@ -12,7 +12,10 @@
   "author": "Petter Kjelkenes <kjelkenes@gmail.com>",
   "exports": {
     ".": "./dist/addon-main.js",
-    "./*": "./dist/*",
+    "./*": {
```

Comment:
this is the `exports` I wrote yesterday:

```
"exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "default": "./dist/index.js"
    },
    "./*": {
      "types": "./dist/*.d.ts",
      "default": "./dist/*"
    }
  },
```

Even the `.` needs their `types` defined (but please check here). And `default` must always be the last key, in the object defining the entry point. I think, this is why it is failing here.

Reply:
thanks!

I updated the `"default"` to come last.

I'm not sure what the types for `"."` would be because right now it's pointing to `"./dist/addon-main.js"` and I don't even see that file in `./dist`